website/releases/2025-06-11-3.0.0-release.md

```diff
@@ -0,0 +1,67 @@
+---
+title: MLflow 3.0.0
+slug: 3.0.0
+authors: [mlflow-maintainers]
+---
+
+# MLflow 3.0
+
+The open source MLflow community has reached a major milestone. Today, we're releasing MLflow 3.0, which brings production-ready generative AI capabilities to the platform that millions of developers trust for ML operations.
+
+This isn't just another feature update. MLflow 3.0 fundamentally expands what's possible with open source ML tooling, addressing the observability and quality challenges that have made GenAI deployment feel like a leap of faith.
+
+## Major Updates
```

Comment:
Do we want to mention the new website layout here?

Reply:
great idea!